client/account-recovery/style.scss

```diff
@@ -0,0 +1,6 @@
+.account-recovery__container {
+	margin: auto;
```

Comment:
Do you need top and bottom to have `auto` margin? Else `margin: 0 auto;` might be more in line with what you're trying to achieve (horizontal centering).